### DISCUSSION TITLE
Is it possible, we can able to recover deleted records?

### DISCUSSION THREAD

**Discussion by komalm** [Sunbird-RC/community#275] [Sunbird RC Technical Specification Feedback]
**Details** - In the RC, That record is not detected in the DB, they soft deleted the record. It means the record is still available in the DB with updated status eg - false or trash

cc @Pratikshakhandagale @jona-r @baljeetsingh

**Comment by parthlawate** [comment]
@tejash-jl can you help with this ?

**Comment by tejash-jl** [comment]
@komalm,
We don't have an API to revert the delete status. You need to do it manually in the DB. You need to update the record's `_status` to `true`
